src/bootstrap/lib.rs

```diff
@@ -328,16 +328,22 @@ pub enum Mode {
     /// Build codegen libraries, placing output in the "stageN-codegen" directory
     Codegen,
 
-    /// Build some tools, placing output in the "stageN-tools" directory.
-    ToolStd,
-    ToolTest,
-    ToolRustc,
+    /// Build some tools, placing output in the "stageN-tools" directory. The
+    /// "other" here is for miscellaneous sets of tools that are built using the
+    /// stage0 compiler in its entirety (target libraries and all). Typically
+    /// these tools compile with stable Rust.
+    ToolOther,
```

Comment:
Could we rename this to `ToolBootstrap` and update the comment above to say "bootstrap compiler" instead of stage0 compiler to avoid the potential ambiguity?